keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverService.kt

```diff
@@ -90,6 +111,7 @@ interface CloudDriverService {
 
   @GET("/aws/images/find")
   suspend fun namedImages(
+    @Header("X-SPINNAKER-USER") serviceAccount: String,
```

Comment:
Would this one be a good candidate for the default?

Reply:
Sometimes we use this where we have the service account, sometimes we use this where we don't have it. I chose not to put the default in here because we can pass in the right one sometimes